loopy/translation_unit.py

```diff
@@ -130,22 +135,6 @@ def map_call(self, expr, expn_state):
 
         return super().map_call(expr, expn_state)
 
-    def map_call_with_kwargs(self, expr, expn_state):
```

Comment:
I think that, generally, these should stay and have their implementations replaced with `raise NotImplementedError`.

Reply:
I agree, replaced all map_call_with_kwargs as said. Also dropped in a link to this PR to provide a bit more context. 78b9a78